pandas/core/generic.py

```diff
@@ -3563,6 +3563,42 @@ def head(self, n=5):
         -------
         obj_head : type of caller
             The first n rows of the caller object.
+
+        See Also
+        --------
+        pandas.DataFrame.tail
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'animal':['alligator', 'bee', 'falcon', 'lion',
+        ...                    'monkey', 'parrot', 'shark', 'whale', 'zebra']})
+        >>> df
+              animal
+        0  alligator
+        1        bee
+        2     falcon
+        3       lion
+        4     monkey
+        5     parrot
+        6      shark
+        7      whale
+        8      zebra
+
+        Viewing the first 5 lines
+        >>> df.head()
```

Comment:
Can you keep a blank line between the explanation and the code line beginning with `>>>` ? (and the same for the other occurences)

This might be needed to satisfy sphinx (not fully sure), but I also think it improves readability